client/components/google-my-business/stats-nudge.js

```diff
@@ -0,0 +1,51 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+
+export const StatsNudge = ( { translate } ) => {
+	return (
+		<div>
+			<SectionHeader
+				className="google-my-business__stats-nudge-header"
+				label={ translate( 'We recommend Google My Business' ) }
+				href="#"
```

Comment:
If it's not a link, why include `href` property?
https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md

Reply:
As you can see here https://github.com/Automattic/wp-calypso/blob/master/client/components/section-header/index.jsx#L19 SectionHeader accepts an `href` prop and if given will end up creating an anchor element in `Card`: https://github.com/Automattic/wp-calypso/blob/master/client/components/card/index.jsx#L56

Given that we'll want this to be clickable and probably link to the GMB settings page I find this a good workable solution to make it behave like a link even though it doesn't link to anything yet

Reply:
I will replace the `SectionHeader` href with an "X" to clarify that clicking it dismisses the card.